ContactPoint.Plugins.CallTools/CallNotifyWindow/CallNotifyWindowService.cs

```diff
@@ -1,80 +1,77 @@
-﻿using System;
+using System;
 using ContactPoint.BaseDesign;
 using ContactPoint.Common;
 using ContactPoint.Common.PluginManager;
 
-namespace ContactPoint.Plugins.CallTools.CallNotifyWindow {
-    internal class CallNotifyWindowService : IService {
+namespace ContactPoint.Plugins.CallTools.CallNotifyWindow
+{
+    internal class CallNotifyWindowService : IService
+    {
         private readonly IPlugin _plugin;
         private readonly CallDelegate _callStatChanged;
-        private bool _isStarted = false;
-        private int _showingWindowTimeout = 0;
 
         public event ServiceStartedDelegate Started;
         public event ServiceStoppedDelegate Stopped;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: The event 'CallNotifyWindowService.Stopped' is never used ([external_roslyn:CS0067](https://sonarcloud.io/organizations/alex-yevtushenko3820/rules#rule_key=external_roslyn%3ACS0067))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=IPP&pullRequest=7&issues=AWfG1-iH8DroMTOoScXB&open=AWfG1-iH8DroMTOoScXB)

[ISSUE_KEY]: AWfG1-iH8DroMTOoScXB